src/components/atoms/MarkdownIconParagraph.js

```diff
@@ -22,10 +22,7 @@ const MarkdownIconParagraph = ({
     <div className={iconClass}>
       {image}
       <p className="iconparagraph__heading"> {icon_heading} </p>
-      <div
-        className="iconparagraph__text"
-        dangerouslySetInnerHTML={{ __html: icon_text }}
-      />
+      <Markdown source={icon_text} escapeHtml={false} />
```

Comment:
Do we lose any of the styling by not reusing the "iconparagraph__text" class here?